### ISSUE TITLE
Force init to throw an error if the user is not a Google+ User

### ISSUE THREAD

**Issue by mohamedmansour** [mohamedmansour/google-plus-extension-jsapi#23]
Right now, if the user is a Google Accounts user, but not a Google+ user, the API doesn''t know about it. We should somehow figure out if the current session is indeed a Google+ user, if not, we should throw an error back within the callback response.

**Comment by ryanpeggs** [comment]
You can use the INITIAL_DATA_API to get the user's Google ID once they have logged in. If you then issue a get request for that user ID (using the PROFILE_GET_API), it will return : 

)]}'

[["er",,,,["op.er","REDIRECTED","/me/createprofile/"]
,200,"op.er"]
]

if the account does not have a public profile. Loading https://plus.google.com/up/accounts/upgrade/ will allow a user to create a profile and the session will become valid after the process has been completed.

**Comment by ryanpeggs** [comment]
Fixed in #31

**Comment by mohamedmansour** [comment]
That is true, but I analyzed intial_data and saw that no public profiles have no ACL, maybe we can assume no ACL = not Google+ user. I am evne wondering that we can remove the get request for plus.google.com and get all the info we need from INITIAL_DATA_API instead.

**Comment by ryanpeggs** [comment]
Using INITIAL__DATA_API with key 1 rather than 14 returns JSON that contains the session authentication. The format is the same as part of the plus.google.com response so you can simply replace the url : plus.google.com with https://plus.google.com/${pagetoken}/_/initialdata?key=1 and the parsing code continues to work.

EDIT: the authentication key is also present using key 14, at the end of the response.

**Comment by mohamedmansour** [comment]
Ah sweet, we will do that then! :)

**Comment by mohamedmansour** [comment]
does this still work?
